lib/issue.js

```diff
@@ -15,8 +15,11 @@ export async function issue({exchanger, exchange} = {}) {
     // run jsonata compiler; only `jsonata` template type is supported and this
     // was validated when the exchanger was created
     const credentials = await Promise.all(credentialTemplates.map(
-      ({template: t}) => jsonata(t).evaluate(variables)));
-
+      async ({template: t}) => {
+        const {vc, ...bindings} = variables;
+        return vc ? jsonata(t).evaluate({vc}, bindings) :
+          jsonata(t).evaluate(variables);
+      }));
```

Comment:
This should just be: `({template: t}) => jsonata(t).evaluate(variables, variables)));`

Note that `vc` should never be elevated to be a "special feature". This system doesn't know what's in the template. Someone could choose to create a template, for example that does: `$eval(bearTorpedo)` and this should still work.

Reply:
Updated [4a0ea8f](https://github.com/digitalbazaar/bedrock-vc-delivery/pull/30/commits/4a0ea8fe05f932f327f6a7611fdc29fe71c0ea26)